test/arrayweeks_test.jl

```diff
@@ -0,0 +1,90 @@
+
+function arrFcomplex(s)
+    # Laplace domain
+    α = complex(-0.3, 6.0)
+    return [1 2;3 4] ./ (s - α)
+end
+
+function Fcomplex(s)
+    # Laplace domain
+    α = complex(-0.3, 6.0)
+    return 1 / (s - α)
+end
+
+@testset "Array functionality" begin
+    @testset "Internal functions" begin
```

Comment:
This testset `"Internal functions"` won't be necessary after the commit https://github.com/elisno/InverseLaplace.jl/commit/bd27190b499bd77ab3d89121156245710c2824d8,
where the Weeks-type can is finally implemented for arrays.